contrib/examples/actions/cloudslang-basic.yaml

```diff
@@ -0,0 +1,13 @@
+---
+description: Greet StackStorm from cloudslang ;) !
+enabled: true
+entry_point: ""
+name: cloudslang-basic
+pack: examples
+parameters:
+  flow_path:
```

Comment:
so basically we bundle CloudSlang with "out of the box" flows (imagine 1 ActionPack CloudSlang is bundled with).
and this flow in included in it.. so no need to add it to ActionPack...